sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -487,6 +499,11 @@ def begin_recognize_content_from_url(self, form_url, **kwargs):
         :keyword list[str] pages: Custom page numbers for multi-page documents(PDF/TIFF), input the number of the
             pages you want to get OCR result. For a range of pages, use a hyphen. Separate each page or
             range with a comma.
+        :keyword str language: The BCP-47 language code of the text in the document.
```

Comment:
Do you know it this is documented somewhere on the service side so we could point to that instead of update the docstring every time a new language is added?

Reply:
Hooray there is some docs! https://docs.microsoft.com/azure/cognitive-services/form-recognizer/language-support